src/modules/dashboard/components/ColonyHome/ColonyTasks/ColonyTasks.css

```diff
@@ -29,3 +29,8 @@
 .noTaskAddition {
   margin-top: 20px;
 }
+
+.taskList {
+  display: flex;
+  max-height: calc(100vh - 350px);
```

Comment:
This one produces the following result:

![Task List](https://user-images.githubusercontent.com/3052635/65904017-286d8d80-e383-11e9-9649-7b1224da7248.gif)